Created a spinning zoom animation with background gradient effect

seanevans-test opened the pull request.

NiallEccles commented:
Thanks for submitting this pull request! I have some changes for you to make but I am unsure if you would complete them in time for the end of Hactoberfest. Here are the changes I've made to your animation:


/**
 * /* By: https://github.com/seanevans-test
 *
 * @format
 */

 @keyframes spinner-zoom {
	0%, 100% { transform: scale(0) rotate(0deg); }
	50% { transform: scale(1) rotate(360deg); }
}

@keyframes backgroundAnimation {
    0%,100% {background-position:0% 47%}
    50%{background-position:100% 54%}
}

.spinning-zoom {
    color: #ff8ae2;
    letter-spacing: 1px;
    text-shadow: 6px 4px 7px #072540;
    font-family: "Inter Bold", "Helvetica", "Arial", san-serif;
    font-weight: 600;
    animation: spinner-zoom 3s infinite;
    /* background: linear-gradient(271deg, #072540, #ff8ae2);
    background-size: 400% 400%;
    animation: backgroundAnimation 3s ease infinite; */
}

You'll notice I've removed the styles you had for the body and title and put them inside your class. This is because when you apply classes to elements that are not your class they interfere with other users animations. I tried to get both of your animations to play on the same element but had no luck. If you do no wish to accept this changes that is perfectly fine, I would just ask you to update this pull request and not include any styles on elements other than your class name!

seanevans-test commented:
Hi, thank you for your feedback, that what you put makes sense, apologies I forgot the impact it would have on the other animations. I've removed the background effect and moved the text h1 styling to also use the .spinning-zoom class.
Hope this looks okay.

seanevans-test commented:
Hi @NiallEccles are you able to approve this MR please :)
